src/components/Sidebar/Sidebar.scss

```diff
@@ -0,0 +1,59 @@
+.app-sidebar {
+  width: 30%;
+  height: 100vh;
+  border-right: 1px solid #ddd;
+}
+
+.app-sidebar-header {
```

Comment:
You should register all styles (for inner elements too) for `.app-sidebar` inside the same class.

`
.app-sidebar {
   .app-sidebar-header { ... }
  .....
}
`